TwitchLib.Api.Core/HttpCallHandlers/TwitchHttpClient.cs

```diff
@@ -120,34 +121,36 @@ public async Task<int> RequestReturnResponseCodeAsync(string url, string method,
             return (int)response.StatusCode;
         }
 
-        private void HandleWebException(HttpResponseMessage errorResp)
+        private async void HandleWebException(HttpResponseMessage errorResp)
```

Comment:
```suggestion
        private async Task HandleWebException(HttpResponseMessage errorResp)
```

`async void` is a big no-no because `void`-returning async methods cannot be awaited and exceptions thrown inside them will crash the application.